diskmap/diskmap.py

```diff
@@ -89,6 +89,17 @@ def __init__(self,
 
         if self.image.shape[0] != self.image.shape[1]:
             raise ValueError("The dimensions of the image should have the same size.")
+        
+        if type(self.image[0,0]) == np.float32:
```

Comment:
This should work, but perhaps something like`if self.image.dtype == np.float32` would be slightly better. Otherwise, I would suggest to use `isinstance`.

Reply:
Agreed. I will make the suggested change.